backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/AccountRelationship.java

```diff
@@ -0,0 +1,37 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "allOf",
+        "by"
+})
```

Comment:
General comment: `@JsonPropertyOrder` is usually used to change the serialization output. In rare occasion it can affect the deserialization but we probably don't need it here.

Reply:
removed. thx